pkgs/development/libraries/cfitsio/darwin-universal.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN cfitsio/configure cfitsio-universal/configure
+--- cfitsio/configure	2018-05-09 21:16:00.000000000 +0200
++++ cfitsio-universal/configure	2018-05-30 14:26:03.000000000 +0200
+@@ -4734,7 +4734,7 @@
+         *)
+             # Build 'Universal' binaries (i386 & x86_64 architectures) and
+             # use rpath token on Darwin 10.x or newer:
+-            SHLIB_LD="$CC -dynamiclib $C_UNIV_SWITCH -headerpad_max_install_names -install_name @rpath/lib\${PACKAGE}.\${CFITSIO_SONAME}\${SHLIB_SUFFIX} -compatibility_version \${CFITSIO_SONAME} -current_version \${CFITSIO_SONAME}.\${CFITSIO_MAJOR}.\${CFITSIO_MINOR}"
++            SHLIB_LD="$CC -dynamiclib -headerpad_max_install_names -install_name @rpath/lib\${PACKAGE}.\${CFITSIO_SONAME}\${SHLIB_SUFFIX} -compatibility_version \${CFITSIO_SONAME} -current_version \${CFITSIO_SONAME}.\${CFITSIO_MAJOR}.\${CFITSIO_MINOR}"
```

Comment:
We don't use rpaths on darwin, using `@rpath/libcfitsio.5.dylib` as the install_name will result in broken binaries/libraries since the stdenv expects us to be using absolute references to libraries instead. 

If a package doesn't do this correctly we generally either patch the build or use install_name_tool to fixup the references at the end of the build. eg.

```nix
{
  postPatch = ''
    substituteInPlace cfitsio-universal/configure \
        --replace "-install_name @rpath/lib" "-install_name $out/lib"
  '';
}
```